sdk/tests/js/tests/tex-image-and-sub-image-utils.js

```diff
@@ -196,17 +198,18 @@ var TexImageUtils = (function() {
    * A fragment shader for a single 3D signed integer texture.
    * @type {string}
    */
-  // Note we always set the tex coordinate t to 0.
+  // Note that the tex coordinate t (the uniform uTCoord) is set to 0.0 by default.
   // Note we always output 1.0 for alpha because if the texture does not contain
   // alpha channel, sampling returns 1; for RGBA textures, sampling returns [0,255].
   var simple3DIntTextureFragmentShaderES3 = [
     '#version 300 es',
     'precision mediump float;',
     'uniform mediump isampler3D tex;',
     'in vec2 texCoord;',
+    'uniform float uTCoord;',
     'out vec4 fragData;',
     'void main() {',
-    '    ivec4 data = texture(tex, vec3(texCoord, 0.0));',
+    '    ivec4 data = texture(tex, vec3(texCoord, uTCoord));',
```

Comment:
The third component should either be r or p (renamed not to confuse with color component r).  Name it t is really confusing.